feature-libs/asm/core/facade/asm-bind-cart.service.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * SPDX-FileCopyrightText: 2022 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Injectable } from '@angular/core';
+import { AsmBindCartFacade, BindCartParams } from '@spartacus/asm/root';
+import { Command, CommandService } from '@spartacus/core';
+import { Observable } from 'rxjs';
+import { AsmConnector } from '../connectors';
+
+@Injectable()
+export class AsmBindCartService implements AsmBindCartFacade {
+  constructor(
```

Comment:
Ideally, I think the bindCart function could have gone in the AsmService class.  However I understand it's more overhead to make it backwards compatible.  Perhaps there were also additional motivations to create a new distinct service class that I'm not aware of. 

It's your call.